src/VaultCredentialRotator/AwsProvider.cs

```diff
@@ -100,10 +99,16 @@ public async Task ExecuteAsync(IConfigurationRoot configuration)
             dynamic awsCredsDeserializedObject = JsonConvert.DeserializeObject(awsCredsContentString);
             var accessKey = (string)awsCredsDeserializedObject.data.access_key;
             var secretKey = (string)awsCredsDeserializedObject.data.secret_key;
+            var leaseID = (string)awsCredsDeserializedObject.lease_id;
+            var leaseDuration = (string)awsCredsDeserializedObject.lease_duration;
 
             Console.WriteLine();
             await SaveCredentialsAsync(accessKey, secretKey);
             Console.WriteLine("Credentials saved!");
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Vault Lease ID: " + leaseID);
+            Console.WriteLine("Vault Token Lease Duration: " + leaseDuration + " seconds");
```

Comment:
Change to use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)